tests/cython/forwardchainer/test_fc.py

```diff
@@ -0,0 +1,52 @@
+import os
+import unittest
+from opencog.forwardchainer import ForwardChainer
```

Comment:
I would suggest merge `backwardchainer` and `forwardchainer` modules into single `rule-engine` module. As they are part of the same rule-engine module. May be do it in separate PR. @noskill, @stellarspot  what do you think?

Reply:
There is some risk to break current code, but as far as I know we have only one usage at the moment.

Reply:
Agreed, especially since the forward and backward chainer might get unified some day.

Reply:
backwardchainer is used in opencog/vqa demo jupyter notebook https://github.com/singnet/dev-portal/tree/master/workshops/docker-opencog

Reply:
I like short names. The guile module (which I didn't create) is called `rule-engine`. However python doesn't accept `-` and `ruleengine` is kinda sucky, so yes maybe `ure` is the best.

Maybe I'll rename the `rule-engine` guile module to `ure` (or keep both names for backward compatibility).